### DISCUSSION TITLE
Snapshot (nightly) builds for all rpiX versions not stored as artefacts at GitHub

### DISCUSSION THREAD

**Discussion by gardian12** [OpenCCU/OpenCCU#2538] [🇺🇸 Q&A]
### Describe the issue you are experiencing

Hello,
I just run for the nightly build images generated by GitHub Action script.
Could not find the rpiX images, while shows the had been created.
Is here my expectation wrong wehere to look for those images?
Or is the GitHub Action script faulty?

I search here for nightly build images -> [Snapshots](https://github.com/jens-maus/RaspberryMatic/actions/workflows/snapshot.yml)

### Describe the behavior you expected

nightly build images are expected to be found here -> [Snapshots](https://github.com/jens-maus/RaspberryMatic/actions/workflows/snapshot.yml)

### Steps to reproduce the issue

1. Well, take a look to -> [Snapshots](https://github.com/jens-maus/RaspberryMatic/actions/workflows/snapshot.yml)

### What is the version this bug report is based on?

3.71.12.20231020 and newer

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

HM-MOD-RPI-PCB

### Anything in the logs that might be useful for us?

```txt
see GitHub logs
```

**Comment by jens-maus** [comment]
Snapshot images are of course placed in a dedicated place unter "Releases". See here:

https://github.com/jens-maus/RaspberryMatic/releases/tag/snapshots

This is however properly documented on the main README of this GitHub project. Simply click on the snapshot badge icon which will route you to the same releases page.

P.S: I Just moved this to the discussion part of this repo since this is definitly no "issue" or "bug" whatsoever. So, please next time read the WARNINGS and suggestion properly before you open an issue ticket which in fact isn't an issue but simply a wrong observation on your end.
